### ISSUE TITLE
web site is only in english

### ISSUE THREAD

**Issue by swirly** [processing/p5.js-website#228]
As a teacher in France, I use p5js to teach basic programming for mys students.

Alas, a lot of them can hardly understand english.

Would be great if the site with multilingual.

As far as I'm concerned, I can produce translation to french.

**Comment by limzykenneth** [comment]
@swirly I believe that's something being worked on at the moment for the new website backend. Not sure what the current status is though. Would be great to have multiple languages for the website and references...

**Comment by lmccart** [comment]
indeed! this is a current priority, and we have an in progress version. i'll be sure to get in touch when we're ready to add translations. we're planning to hold some kind of translatathon event later this year with remote participants, hope you can join!

**Comment by swirly** [comment]
Sure. Will be glad to do it !

**Comment by chiunhau** [comment]
Hi @lmccart , please contact me if you need any help with Chinese translation 😃 I'll be glad to contribute because I've held many workshops features p5.js in Taiwan, but our students (mostly under 18) were not familiar with English doc though.

**Comment by lmccart** [comment]
thanks @chiunhau! I'll be sure to get in touch with you when we're ready, this is a project I'm focusing on this summer.

**Comment by aamontoya89** [comment]
Hi!
I'm currently working over the summer on internationalization of p5.js in different areas:
- researching on the best way to make the p5.js website international, building on top of the work done here: https://github.com/mayaman/p5js-website
- preparing workshops to be done in Spanish, particularly in Chile
- translating material for reference, already completed the first draft of the make p5.js book

i'm documenting all of my work here https://github.com/aamontoya89/p5_spanish
anyone interested in collaborating and discussing with me would be great, particularly
on experiences doing workshops in other languages and thoughts and tools for internationalization
of the website

best =)

**Comment by limzykenneth** [comment]
@aamontoya89 Used to to localisation QA so do let me know. I can do both simplified and traditional Chinese. 😃
